docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,373 @@
+| all_equal | All requested principals must be equal to the set string. If the set string is `foo.example.com`, host certificates limit access only to `foo.example.com`. | `all_equal(host_cert.principals, "foo.example.com")` |
+| all_end_with |  All requested principals must end with the set string. If the set string is `.example.com`, host certificates limit access only to `*.example.com`. | `all_end_with(host_cert.principals, ".example.com")` |
+| is_subset | All requested principals must be contained in the listed set. | `is_subset(host_cert.principals, "foo.example.com", "bar.example.com")` |
+
+In the case of an example role for use within this guide, use a text editor of your choice to create a role YAML named `tbotrole.yaml` with the following configuration options, replacing `nodename.my.domain.com` with your own domain name,
+or the domain associated with your cluster:
+
+```code
+ kind: role
+metadata:
+  name: hostcert-bot
+version: v5
+spec:
+  allow:
+    rules:
+      - resources:
+          - host_cert
+        verbs:
+          - create
+        where: 'is_subset(host_cert.principals, "nodename.my.domain.com")'
+```
+
+Once complete, create your role to apply to the tbot configuration in the next step:
+
+```code
+$ tctl create tbotrole.yaml
+```
+
+
+### Step 3/4 Create Your `tbot` Configuration
+
+Before you create a bot user, you need to determine which role(s) you want to
+assign to it. You can use the `tctl` command below to examine what roles exist
+on your system.
+
+<ScopedBlock scope={["cloud"]}>
+On your client machine, log in to Teleport using `tsh`, then use `tctl` to examine
+what roles exist on your system.
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+Connect to the Teleport Auth Server and use `tctl` to examine what roles exist on
+your system.
+</ScopedBlock>
+
+```code
+$ tctl get roles --format=text
+```
+
+You will see something like the output below on a fresh install of Teleport with the
+default roles—your cluster may have different roles. In this example, let's
+assume you want to give the bot the `hostcert-bot` role to allow it to generate host certificates.
+
+```
+Role    Allowed to login as                           Node Labels Access to resources
+------- --------------------------------------------- ----------- ----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read
+editor
+hostcert-bot  host_cert:create,list,read,update,delete             user:list,create,read,update,delete,...
+```
+
+`tbot` can join the Teleport Cluster using a number of supported methods. This guide will highlight both the `token` and the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+Note that the following commands may be entered on any Teleport node with `tctl` access to the cluster.
```

Comment:
We cover this in the `tctl.mdx` partial. If we do want to keep this sentence, I think this wording can be clearer. What does "Teleport node" mean? Can it include a user's workstation?